tests/unit/services/migration/HomeAuthXmlMigrationTest.php

```diff
@@ -0,0 +1,144 @@
+<?php declare(strict_types = 1);
+namespace PharIo\Phive;
+
+use PharIo\FileSystem\Directory;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \PharIo\Phive\HomeAuthXmlMigration
+ */
+class HomeAuthXmlMigrationTest extends TestCase {
+    use MigrationMocks;
+
+    protected function tearDown(): void {
+        @\unlink(__DIR__ . '/tmp/auth.xml');
+        @\unlink(__DIR__ . '/tmp/phive-auth.xml');
+        @\unlink(__DIR__ . '/tmp/phive-auth.xml.backup');
+        parent::tearDown();
+    }
+
+    /**
+     * phive-auth.xml exists, and auth.xml exists
+     */
+    public function testInError(): void {
```

Comment:
I like the test implementation but why put the actually useful test name into a docblock and use a rather generic method name?

I think we should rename the test to something like `testMigrationFailsWhenOldAndNewAuthFileExist`.

Same goes for the other test methods.